ecl/resource_ecl_mlb_load_balancer_v1.go

```diff
@@ -15,20 +17,39 @@ import (
 func reservedFixedIPsSchemaForResource() *schema.Schema {
 	return &schema.Schema{
 		Type:     schema.TypeList,
-		Required: true,
-		MinItems: 4,
+		Optional: true,
+		Computed: true,
 		MaxItems: 4,
 		Elem: &schema.Resource{
 			Schema: map[string]*schema.Schema{
 				"ip_address": &schema.Schema{
 					Type:     schema.TypeString,
-					Required: true,
+					Optional: true,
+					Computed: true,
 				},
 			},
 		},
+		DiffSuppressFunc: suppressReservedFixedIPsDiff,
 	}
 }
 
+func suppressReservedFixedIPsDiff(k, old, new string, d *schema.ResourceData) bool {
+	if strings.HasSuffix(k, ".#") {
+		oldCount, errOld := strconv.Atoi(old)
+		newCount, errNew := strconv.Atoi(new)
+
+		if errOld != nil || errNew != nil {
+			return false
+		}
+
+		if oldCount == 4 && newCount <= 3 {
```

Comment:
[IMO]
`MaxItems: 4` との関連を想起しやすい ↓ の方が好みですが、修正の是非はお任せします!!

```suggestion
		if oldCount == 4 && newCount < 4 {
```

Reply:
特にこだわりはないので `< 4` に修正しました